frontend/src/components/ErrorDisplay/ErrorDisplay.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Alert } from 'reactstrap';
+import { CLEAR_ERRORS } from '../../flux/actions/types';
+
+const ErrorDisplay = () => {
+  const error = useSelector(state => state.error.message.message);
```

Comment:
@marcianszlovak are you sure this is good?
if there's no error it'll give an error

Reply:
so if there's a test for it with error message and without it should've failed and showed you that there's a bug